### PR TITLE
Oxidation of remote manifests

### PR DESCRIPTION
**Pull request by Max-7**
Depends on #2048

### CODE REVIEW

**Review by touilleMan — APPROVED** [review]
lgtm

File: parsec/api/data/entry.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ class EntryID(UUID4):
 
 class EntryName(StrBased):
     # Ignore the REGEX
-    REGEX = None
+    REGEX = re.compile(".*")
```
Comment (touilleMan) [inline parsec/api/data/entry.py:37]: Should modify typing for `StrBased` to `Optional[Pattern[str]]` instead

Comment (Max-7) [reply parsec/api/data/entry.py]: Fixed

File: parsec/core/mountpoint/winfsp_operations.py [hunk 2]
```diff
@@ -363,6 +365,11 @@ def read_directory(self, file_context, marker):
         # NOTE: The "." and ".." directories should ONLY be included
         # if the queried directory is not root
 
+        print(
+            f"READ DIR marker: {marker}, file ctx: {file_context.path!r} (is_root: {file_context.path.is_root()})"
+        )
+        print(f"READ DIR stats: {stat!r}")
+
```
Comment (vxgmichel) [inline parsec/core/mountpoint/winfsp_operations.py:372]: Debug prints

**Pull request merged**